src/config/config.py

```diff
@@ -39,6 +39,9 @@
 	},
 	'Debian 10': {
 		'Buster': 'Debian_Buster_List.json'
+	},
+	'ClefOS': {
+		'ClefOS Epel 7': 'ClefOS_7_List.json'
```

Comment:
Epel is a different section of the repository :smile: (we may include it later, based on feedback from Neale) according to the script above, you want "base" here.